numpy/core/src/umath/ufunc_object.c

```diff
@@ -2230,7 +2246,9 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
     /* Get all the arguments */
     retval = get_ufunc_arguments(ufunc, args, kwds,
                 op, &order, &casting, &extobj,
-                &type_tup, &subok, NULL, &axes, &keepdims);
+                &type_tup, &subok, NULL, &axes, &axis, &keepdims);
+    /* if given, need our own reference to axes, as we XDECREF it at the end */
```

Comment:
Why not just leave it alone then? This doesn't seem to justify doing either at the moment.

Reply:
I'll edit the comment to state this explicitly.